Library/ENV/scm/git

```diff
@@ -1,4 +1,10 @@
-#!/usr/bin/ruby -W0
+#!/bin/sh
```

Comment:
We're not using any `bash` features in this very short snippet, so I think `/bin/sh` is a more suitable choice. I haven't benchmarked this, but I suspect it also helps to keep the footprint of these wrappers as small as possible: `bash` when invoked as `sh` switches to traditional `sh`/POSIX mode and thus looks for fewer files to read during startup.